### PR TITLE
Prevent app from crashing if mongodb isn't running

### PR DESCRIPTION
**Pull request by phuvu0107**

### CODE REVIEW

**Review by evanugarte — APPROVED** [review]
I'm with it

**Review by neelp03 — APPROVED** [review]
me too

File: server.js [hunk 1]
```diff
@@ -51,17 +51,19 @@ const startBot = async () => {
...
+      poolSize: 50,
+      bufferMaxEntries: 0,
+      keepAlive: 120,
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+      useCreateIndex: true,
+      user: DATABASE_USER,
+      pass: DATABASE_PASSWORD,
+    })
+    .catch(() => console.log('Unable to connect to MongoDB', DATABASE_URL));
```
Comment (evanugarte) [inline server.js:66]:
```suggestion
    .catch(() => console.log('Unable to connect to MongoDB at', DATABASE_URL));
```

**Comment by akshtshrma24** [comment]
nice very good 
10/10

**Review by akshtshrma24 — COMMENTED** [review]
nice

**Review by akshtshrma24 — APPROVED** [review]
approve

**Pull request merged**